src/AcmAlert/AcmAlert.tsx

```diff
@@ -1,17 +1,31 @@
-import React from 'react'
-import { Alert, AlertGroup, AlertProps } from '@patternfly/react-core'
+import React, { useState } from 'react'
+import { Alert, AlertActionCloseButton, AlertGroup, AlertProps } from '@patternfly/react-core'
 import { AlertGroupProps } from '@patternfly/react-core/dist/js/components/AlertGroup/AlertGroup'
+import { makeStyles } from '@material-ui/styles'
 
 type AcmAlertProps = AlertProps & {
     subtitle?: string | React.ReactNode
+    noClose?: boolean
 }
 
+const useStyles = makeStyles({
+    alert: {
+        '--pf-c-alert--BoxShadow': '0 1px 2px 0 rgba(0, 0, 0, 0.1)',
+        '--pf-c-alert__icon--MarginRight': '1rem',
+    },
+})
+
 export function AcmAlert(props: AcmAlertProps) {
-    return (
-        <Alert {...props}>
+    const classes = useStyles()
+
+    const { noClose, ...otherProps } = props
+    const [showAlert, setShowAlert] = useState(true)
+    const closeAlert = !noClose && <AlertActionCloseButton onClose={() => setShowAlert(!showAlert)} />
```

Comment:
It's not really a toggle, so might be clearer to just say that it always sets `showAlert` to false.

```suggestion
    const closeAlert = !noClose && <AlertActionCloseButton onClose={() => setShowAlert(false)} />
```

Reply:
good catch, thanks!